common/src/character_controller.rs

```diff
@@ -107,7 +107,14 @@ impl CharacterControllerPass<'_> {
         let ray = graph_collision::Ray::new(math::origin(), displacement_normalized);
         let tanh_distance = displacement_norm.tanh();
 
-        let cast_hit = graph_collision::sphere_cast(self.position, &ray, tanh_distance);
+        let cast_hit = graph_collision::sphere_cast(
+            self.cfg.character_radius,
+            self.graph,
+            &ChunkLayout::new(self.cfg.chunk_size as usize),
```

Comment:
Should we just store the `ChunkLayout` in the config structure directly?

Reply:
Possibly. The idea of the `ChunkLayout` struct is that it would eagerly pre-compute things that depend on the dimension, just like `dodeca` pre-computes things that don't depend on the dimension. Using it more widely would have the additional benefit of allowing experimentation with nonlinear layouts where cube-corner voxels are made a bit smaller, and cube-center voxels are made a bit larger.

This design choice has far-reaching implications, and I'm not sure whether it's within scope for this PR. I had decided to put the seam between the current approach and this new `ChunkLayout` approach here, but if you're a fan of this idea, I can add it to `sim_config` in place of `chunk_size`.

Reply:
Let's err on the side of avoiding scope creep.